src/Hero.java

```diff
@@ -0,0 +1,102 @@
+public class Hero extends Entity {
+
+    private int currentLevel = 1;
+    private double currentXp = 0;
+    private double xpPerLevel;
+    private int dmgIncreasePerLevel;
+    private int hpIncreasePerLevel;
+    private double cooldownMultiplierPerLevel;
```

Comment:
final?

Reply:
xpPerLevel-t is? Az ötletem az volt hogy szintenként növekedik az xp követelmény.

Reply:
A többit szimplán elfelejtettem. Köszi, hogy rámutattál.

Reply:
Nekem szerintetm nem volt ilyen kovetelmenyem, de nem zavar ha beleteszitek. (kesobb lattam azt a reszt)